builtin/game/item.lua

```diff
@@ -147,6 +147,23 @@ function core.wallmounted_to_dir(wallmounted)
 	return wallmounted_to_dir[wallmounted]
 end
 
+function core.dir_to_yaw(dir)
+	if dir.z < 0 then
+		return math.pi - math.atan(dir.x / dir.z)
+	elseif dir.z > 0 then
+		return -math.atan(dir.x / dir.z)
+	elseif dir.x < 0 then
+		return math.pi
+	else
+		return 0
+	end
+end
```

Comment:
Never knew it existed - I'm double checking I can use it (will need to account for the axis transformation) :)